_episodes/16-resources.md

```diff
@@ -97,8 +97,7 @@ Some interesting fields include the following:
 
 ## Measuring the statistics of currently running tasks
 
-One very useful feature of SLURM is the ability to SSH to a node where a job is running and check
-how it's doing. To do this, check where a job is running with `squeue`, then run `ssh nodename`.
+Typically, clusters allow users to SSH directly into worker nodes from the head node. This is useful to check on a running job and see how it's doing. To do this, check where a job is running with `squeue`, then run `ssh nodename`.
 
```

Comment:
should we really encourage people to ssh into nodes?

Reply:
Maybe not "encourage", but it can be a useful diagnostic tool, it seems reasonable to me that learners should be aware of it. We could add some cautionary text about only doing it for diagnostic purposes, and not as a way of cheating the scheduler.

Reply:
that's exactly my concern. let's pack it into a call-out for now as a compromise. In the end, all users need to have ssh access to all compute nodes as e.g. MPI uses that to administer the processes.

Reply:
I've put in a call-out about SSH, but the coordination with the following content is now fairly poor, that section talks about running "top" and "free" to see what's going on, but this only makes sense on the node, which we are now advising against?
This needs more attention.